### PR TITLE
Docs: add doc about oracle

### PR DESCRIPTION
**Pull request by chaen**
just few lines of docs

### CODE REVIEW

File: docs/50_troubleshoot.md [hunk 1]
```diff
@@ -96,4 +96,8 @@ The packages that are in `manualDependencies` are added to the dependencies that
 
 ## LDD check failing
 
-The `check_ldd` test looks at the dependencies of the binaries in DIRACOS. It finds those pointing outside of DIRACOS (can be due to rpath) and those not found. There is a list of known "broken" dependencies (`tests/integration/knownMissingDependencies.txt`). Ideally, this list should be empty, but it will never be the case. So if you cannot do differently, you can always add your libraries there.
\ No newline at end of file
+The `check_ldd` test looks at the dependencies of the binaries in DIRACOS. It finds those pointing outside of DIRACOS (can be due to rpath) and those not found. There is a list of known "broken" dependencies (`tests/integration/knownMissingDependencies.txt`). Ideally, this list should be empty, but it will never be the case. So if you cannot do differently, you can always add your libraries there.
+
+## About Oracle
+
+Because of licenses reasons we cannot distribute the Oracle RPMs. Thus, if you want to use the `cx_Oracle` python library packaged in DIRACOS, you should install `oracle-instantclient-basic` and `oracle-instantclient-devel` packages on your server.
```
Comment (chrisburr) [inline docs/50_troubleshoot.md]: Just for my benefit, is oracle used anywhere other than the LHCb bookkeeping?

Comment (petricm) [reply docs/50_troubleshoot.md]: We don't use it

Comment (chaen) [reply docs/50_troubleshoot.md]: it's true that we could potentially move it to `LHCbDIRACOS`. But I don't know if anyone else uses it

**Comment by chaen** [comment]
So what should I do with that ? Move cx_Oracle to LHCbDIRACOS ?

**Comment by petricm** [comment]
Make it so.

**Comment by fstagni** [comment]
Oracle is today said to be supported by vanilla DIRAC. I believe that at the same time LHCb is the only user. If we remove it from DIRACOS, we remove it from Vanilla DIRAC code and we stop saying it's supported. I will take this at the next BiLD (this week).

**Comment by petricm** [comment]
Given the discussion last Thursday, we can remove Oracle.

**Comment by chrisburr** [comment]
If we're saying Oracle "support" isn't removed until `v7r1` should we wait a little before removing it from DIRAC OS?

**Comment by petricm** [comment]
Given that only LHCb is using it, despite this being a huge change, it should go with some major release of DIRAC, but then we would have to have multiple latest versions of DIRACOS, which is also just an overhead, which nobody would take advantage of. I would drop it immediately...

**Comment by fstagni** [comment]
I agree to drop immediately. And, TBH, the very few installations that moved to v7r0 are not going to see any difference. It's not elegant nor, in a way, correct, but after also the discussion in the last BiLD I wouldn't make a fuss.

**Pull request closed without merging**